src/data.table.h

```diff
@@ -63,6 +63,11 @@
 #define NEED2UTF8(s) !(IS_ASCII(s) || (s)==NA_STRING || IS_UTF8(s))
 #define ENC2UTF8(s) (!NEED2UTF8(s) ? (s) : mkCharCE(translateCharUTF8(s), CE_UTF8))
 
+// Types used to dynamically limit DT threads when passing to num_threads
+#define OMP_BATCH 2 // limit to number of iterations (batches)
```

Comment:
is there some tradeoff of using macro vs `enum`?

Reply:
macro is simpler